docs/hypernode-platform/nginx/hypernode-managed-vhosts.md

```diff
@@ -116,11 +116,11 @@ This command will not only request a LE Certificate but because of the `--force-
 
 ## Varnish and Hypernode Managed Vhosts
 
-Using Varnish in combination with Varnish works slightly different with HMV enabled. Of course Varnish needs to be enables with the **systemctl** tool. `hypernode-systemctl settings varnish_enabled True`
+Using Varnish works slightly different with HMV enabled. Of course Varnish needs to be enables with the **systemctl** tool. `hypernode-systemctl settings varnish_enabled True`
```

Comment:
Typo: `enables -> enabled`

Reply:
Maybe we should also call it `Hypernode Managed Vhosts ` instead of `HMV`. Perhaps not all our customers doesnt understand what `hmv` means

Reply:
Fixed the 'enabled' and also some others I noticed.

About the HMV thing, this particular document page is not that long and it makes it very clear what HMV means, multiple times. I think there's 5 occurrences of "Hypernode Managed Vhosts (HMV)" in the page so I don't think that's an issue